src/ast/converter.js

```diff
@@ -32,7 +32,7 @@ import { AggregateType } from '../aggregate';
 import moment from 'moment-timezone';
 
 const MAX_DISTINCT_VALUES = 1000;
-const MAX_TILE_RECORDS = 1000;
+const MAX_TILE_RECORDS = 5000;
```

Comment:
I know I said this during standup, but lets put this back to 1000, and then we'll set the value inside of the rails app directly (and from a chart config value) so that it can be changed quickly if necessary